libslide/src/bignum/utils.rs

```diff
@@ -64,6 +97,27 @@ pub fn ifft(item: Vec<Complex>, n: usize) -> Vec<Complex> {
     a
 }
 
+// converts a multi digit polynomial to an single digit vector
```

Comment:
Can you add an example in the doccomments of this function?

Reply:
so wait  how do I do contribute to our rust docs beyond the triple / or is that the way to do it

Reply:
Just triple /. you can also add code blocks in triple backticks (```) and include markdown